app/Models/Bible/BibleFileset.php

```diff
@@ -514,13 +520,11 @@ public function scopeConditionToExcludeOldDA16Format(Builder $query) : Builder
                     ->whereColumn('bfcaudio.set_type_code', '=', $from_table.'.set_type_code')
                     ->where(
                         DB::raw(\sprintf(
-                            'SUBSTRING(%s.id, %d, %d)',
-                            $from_table,
-                            strlen(self::V1_SUFIX_AUDIO_16_KBPS)*-1,
-                            strlen(self::V1_SUFIX_AUDIO_16_KBPS)
+                            'CHAR_LENGTH(%s.id)',
+                            $from_table
                         )),
                         '=',
-                        self::V1_SUFIX_AUDIO_16_KBPS
+                        self::V1_AUDIO_16_KBPS_FILESET_LENGTH
```

Comment:
@bradflood , I made this change to the code because it was taking too long to run. The performance was being affected by the SUBSTRING method, which was taking a long time to process each row. This was causing issues with the following postman test:

https://digitalbibleplatform.postman.co/workspace/DBP4-workspace~0a283f7f-06d9-467d-97f0-fef92d18bd3d/request/12519377-c0d1ac98-aa6b-4394-9803-9e23e87d6101